packages/components/src/Lettera/Models.purs

```diff
@@ -225,12 +226,8 @@ type Author =
   , email  :: Maybe String
   }
 
--- TODO: Could be a type class
-encodeStringifyArticle :: Article -> String
-encodeStringifyArticle = stringify <<< encodeJson <<< articleToJson
-
-encodeStringifyArticleStubs :: Array ArticleStub -> String
-encodeStringifyArticleStubs = stringify <<< encodeJson <<< map articleStubToJson
+articleToArticleStub :: Article -> ArticleStub
+articleToArticleStub = unsafeCoerce
```

Comment:
Why `unsafeCoerce`? Wouldn't it lead to horrible bugs if we change the representation of either type?

Reply:
Convenience, execution speed and the way Article and ArticleStub are implemented in Lettera we'd have to do something very special to have them diverge in other fields besides body.

But I changed it to have assignments, field by field. If `Article` and `ArticleStub` had all the fields defined it could be done with just `Record.delete (Proxy :: Proxy "body")` (which, under the hood, is just `unsafeCoerce`).